test/runnable/test2968.d

```diff
@@ -0,0 +1,24 @@
+// DISABLED: win linux freebsd dragonflybsd netbsd
+
+struct __CFArray; //try to call some CF functions with strings
+alias CFArrayRef = __CFArray*;
+alias CFIndex = long;
+
+extern(Objective-C) CFArrayRef CFArrayCreate(void* allocator, const void** values, long numValues, void* cbs);
+extern(Objective-C) CFIndex CFArrayGetCount(CFArrayRef theArray);
+extern(Objective-C) const(void *) CFArrayGetValueAtIndex(CFArrayRef theArray, CFIndex idx);
+
+pragma(framework, "Foundation");
```

Comment:
* The above functions are part of the `CoreFoundation` framework

* I think the pragma should be at the top of the module